cli/bench_command.go

```diff
@@ -489,12 +516,12 @@ func jsPublisher(c benchCmd, nc *nats.Conn, progress *uiprogress.Bar, msg []byte
 func kvPutter(c benchCmd, nc *nats.Conn, progress *uiprogress.Bar, msg []byte, numMsg int, offset int) {
 	js, err := nc.JetStream()
 	if err != nil {
-		log.Fatalf("Couldn't get the JetStream context: %v", err)
+		log.Fatalf("Couldn't get the JetStream context %v", err)
 	}
 
 	kvBucket, err := js.KeyValue(c.subject)
 	if err != nil {
-		log.Fatalf("Couldn't find kv store %s: %v", c.subject, err)
+		log.Fatalf("Couldn't find kv store %s %v", c.subject, err)
```

Comment:
Generally people would write go errors in a way that they would make sense after “:”. Some might be weird without. Especially some that are a chain or “:” sep errors. 

Agree though here where it’s user shown the : thing is annoying. But I found it weird both ways :)

Reply:
I changed this because the errors coming back from nats.go start with 'nats: xxxxx' so if you want to add some context ahead of the nats.go error (like here repeating the subject) and end that with a ':' you end up with a chain indeed (.e.g. 'nats: error: nats: bucket not found') which looked weird  and annoying to me so I took mine out. Didn't realize it was the convention to have this chain of : to delineate the layers of the onion and therefore you must always end on a :  so reverting to doing that.